src/background/actions/action-payloads.ts

```diff
@@ -128,6 +129,10 @@ export interface SetBugServicePayload extends BaseActionPayload {
     bugServiceName: string;
 }
 
+export interface SaveBugFilingSettingsPayload extends SetBugServicePayload {
```

Comment:
Should this be Save_Issue_FilingSettingsPayload given that the message is `SaveIssueFilingSettings`

Reply:
We have a work item (1506355) to rename all at once. In this PR there are several places that I don't really know which one should I use at present.